config.yml

```diff
@@ -304,6 +311,43 @@ jobs:
       - deploy-to-pantheon-env:
           environment: 'live'
 
+
+  security_updates_check:
+    <<: *defaults
+    docker:
+      # quay.io/pantheon-public/build-tools-ci:8.x-php${PHPVERSION}
+      - image: quay.io/pantheon-public/build-tools-ci:8.x-php<< pipeline.parameters.php-version >>
```

Comment:
If you don't need terminus, using  straight defaults here would be best. See the static tests job for an example.

Reply:
I'm going to keep the image because We are going to change it anyway in the near future when we implement the Config export